### PR TITLE
 about dialog blemishes

### PR DESCRIPTION
**Pull request by casperlamboo**
Cura-11263

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -152,6 +152,25 @@ def _conan_installs(self):
             }
         return conan_installs
 
+    def _python_installs(self):
+        python_installs = {}
+
+        # list of python installs
+        outer = '"' if self.settings.os == "Windows" else "'"
+        inner = "'" if self.settings.os == "Windows" else '"'
```
Comment (wawanbreton) [inline conanfile.py:160]: Is that for Mac ? Linux seems to support the 2 syntaxes. I will trust you if you just tell me that is IS required...

**Review by wawanbreton — COMMENTED** [review]
Looks good, I'll make a build to check that it does work, at least for linux

File: conanfile.py (hunk 1 above)
Comment (casperlamboo) [reply conanfile.py:160]: apparently for mac it also works both ways, I'm going to change it to the windows case and if it installs correctly on my mahcine and it won't break the conan package it should work every where 👍

Comment (casperlamboo) [reply conanfile.py]: https://github.com/Ultimaker/Cura/pull/17186/commits/51ab7409058fd38c4541064d93e87fd64902872f

**Pull request merged**